v2/emailpassword/migration/account-creation/ep-migration-without-password-hash.mdx

```diff
@@ -0,0 +1,175 @@
+---
+id: ep-migration-without-password-hash
+title: User Creation without password hahes
+hide_title: true
+---
+
+# User Creation without password hahes
+
+
+:::caution
+The recommended method for migrating users to SuperTokens is by [importing users with their password hashes](./user-creation).
+You should only use the following method if you do not have access to your user's password hashes and still have access to your previous identity provider.
+:::
+
+Supertokens also supports the "**just in time**" user migration strategy for when password hashes cannot be exported from your legacy provider.
+In this approach, the login flow is modified so that if a legacy user tries to authenticate, SuperTokens will check if the user exists in the legacy provider and whether the input credentials are valid.
+If the user exists in the leagacy provider and the credentials are valid, SuperTokens will retrieve the user data from the legacy provider and create a corresponding SuperTokens user account. 
```

Comment:
```suggestion
In this approach, the login flow is modified so that if a legacy user tries to authenticate, SuperTokens will check if the user exists in the legacy provider and whether the input credentials are valid and if so, SuperTokens will retrieve the user data from the legacy provider and create a corresponding SuperTokens user account. 
```